kuma/wiki/views/revision.py

```diff
@@ -84,19 +84,15 @@ def compare(request, document_slug, document_locale):
     The ids are passed as query string parameters (to and from).
     """
     locale = request.GET.get('locale', document_locale)
+    if 'from' not in request.GET or 'to' not in request.GET:
+        raise Http404
```

Comment:
FWIW the 404 was there already, but I get what you mean. 422 seems like an odd choice though as it has little to no precedence in Django land and comes from the sub protocol WebDAV. Also, I don't freaking know how the load balancer treats 422s, so 404 will have to do for now.